### PR TITLE
Create timetable view webpage

### PR DESCRIPTION
**Pull request by keyruhn**
Current progress on timetable view file

### CODE REVIEW

**Comment by keyruhn** [comment]
All of the current progress on the front-end of CourseFlow

File: Front-End/assets/blank.txt
Comment (Vortrox) [inline Front-End/assets/blank.txt:1]: What's this for?

Comment (Vortrox) [reply Front-End/assets/blank.txt:1]: Maybe we should remove this if it's not being used

**Review by Vortrox — CHANGES_REQUESTED** [review]

**Comment by Vortrox** [comment]
Merging for end of trimester handover

**Review by Vortrox — APPROVED** [review]

**Pull request merged**